collectors/ebpf.plugin/README.md

```diff
@@ -186,11 +192,19 @@ attaches `kprobes` for kernel functions:
 
 #### Dirty pages
 
+Pages in the page cache modified after being brought in are called “dirty pages.” Since non-dirty pages in the page cache
+ have identical copies in secondary storage (e.g. hard disk drive or solid-state drive), discarding and reusing their space
+ is much quicker than paging out application memory, and is often preferred over flushing the dirty pages into secondary storage
+ and reusing their space.
+
 On `cachestat_dirties` Netdata demonstrates the number of pages that were modified. This chart shows the number of calls
 to the function `mark_buffer_dirty`.
 
 #### Page cache hits
```

Comment:
I would explain this section as the first, then dirty pages, and then the page cache ratio.

Reply:
I adjusted the order.